src/server/user-controller.js

```diff
@@ -281,11 +283,19 @@ class UserController {
     return res.status(201).send({ roles: newRoles });
   }
 
-  async handlePostUserLockedOut(req, res) {
+  async handlePostUserAccountLockedOnBodySchema(req, res) {
```

Comment:
Oops, that renaming went wrong :thinking:

Reply:
I know it looks wrong, but it actually isn't :( handle_post_userAccountLockedOn_body_schema, cause `accountLockedOn` is the field name on the `user`.

Reply:
But this is the handler function's name, it cannot end with `Schema` :thinking:

Reply:
Oh of course it cannot! Merry Christmas 🎅 😅 
Dropped the `BodySchema` postfix, thanks!

Reply:
Merry ChristmasSchema :mx_claus: :rofl:

Reply:
And even Christmas**Body**Schema sadly 😂